scripts/github_issue_template.tmpl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 NXP Corp.
+# SPDX-License-Identifier: Apache-2.0
+'''
+    git hub issue template
+'''
+
+GITHUB_ISSUE_TITLE_TEMPLETE = '''
+tests-ci :${CASE_NAME} : ${ZEPHYR_VERSION}: ${PLATFORM}: test ${RESULT}
+'''
+
+GITHUB_TEMPLETE = '''
+**Describe the bug**
+${CASE_NAME} test is ${RESULT} on ${ZEPHYR_VERSION} on ${PLATFORM}
+
+see logs for details
+
+**To Reproduce**
+1. 
+```
+scripts/twister --device-testing --device-serial /dev/ttyACM0 -p ${PLATFORM} \
+--testcase-root tests --sub-test ${APP_NAME}
```

Comment:
no need for pointing to tests with --testcase-root. It also can be incorrect if it is a sample that failed.

Reply:
ok, remove the --testcase-root